specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2018-03-31/managedClusters.json

```diff
@@ -309,6 +309,39 @@
             "required": true,
             "type": "string",
             "description": "The name of the managed cluster resource."
+          },
+          {
+            "name": "format",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "enum": [
+              "azure",
+              "exec"
+            ],
+            "x-ms-enum": {
+              "name": "Format",
```

Comment:
just curious, why `Format` is Title case, while `login` is all small, this inconsistent is little confusing